src/solvers/refinement/README.md

```diff
@@ -6,59 +6,92 @@
 
 \section string_solver_interface String solver interface
 
+The string solver is particularly aimed at string logic, but since it inherits
+from other solvers it is also capable of handling arithmetic, array logic,
```

Comment:
`other solvers` is `bv_pointerst`. Ideally, there should be a cross-ref to the documentation of solvers/flattening (even if that one is just a stub right now).